src/io/react-native/react-native-worker.ts

```diff
@@ -172,13 +172,15 @@ async function makeIo(): Promise<EdgeIo> {
       uri: string,
       opts?: EdgeFetchOptions
     ): Promise<EdgeFetchResponse> {
-      return await window.fetch(uri, opts)
-    },
+      const { avoidCors = 'auto' } = opts ?? {}
+
+      if (avoidCors === 'never') {
+        return await nativeFetch(uri, opts)
+      }
+      if (avoidCors === 'always') {
+        return await window.fetch(uri, opts)
+      }
```

Comment:
I think these are backwards too. Saying "never avoid CORS" means using the browser, and saying "always avoid CORS" means using the native fetch function.